published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+# IPFS Weekly #5
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the February sprints:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+  - [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated.  
+  - [@noffle](//github.com/noffle), [@whyrusleeping](//github.com/whyrusleeping) and [@chriscool](//github.com/chriscool) have been doing lots to make our tests greener. This'll let us get PRs in faster because we won't need to wonder whether our CI is lying to us or not. This gives us more confidence about the code we ship, and allows us to ship more often.
```

Comment:
There are ways to say this that give context accurately. The current language is inaccurate (CI isn't lying, the tests are wrong) and makes the CI or testing work look worse than it is. This is a common thing many teams struggle with and must fix. maybe this:

- @noffle, @whyrusleeping and @chriscool have been improving the test suite, fixing flaky tests which used to fail inaccurately. This will let us merge PRs faster, as CI should correctly categorize success and failure.